cmd/param.go

```diff
@@ -41,22 +41,43 @@ func init() {
 	paramDiffCmd.PersistentFlags().String(flagParamComponent, "", "Specify the component to diff against")
 }
 
+var paramShortDesc = map[string]string{
+	"set":  `Change component or environment parameters (e.g. replica count, name)`,
+	"list": `List known component parameters`,
+	"diff": `Display differences between the component parameters of two environments`,
+}
+
 var paramCmd = &cobra.Command{
 	Use:   "param",
-	Short: `Manage ksonnet component parameters`,
-	Long: `Parameters are the customizable fields defining ksonnet components. For
-example, replica count, component name, or deployment image.
-
-Parameters are also able to be defined separately across environments. Meaning,
-this supports features to allow a "development" environment to only run a
-single replication instance for it's components, whereas allowing a "production"
-environment to run more replication instances to meet heavier production load
-demands.
-
-Environments are ksonnet "named clusters". For more information on environments,
-run:
-
-    ks env --help
+	Short: `Manage ksonnet parameters for components and environments`,
+	Long: `
+Parameters are customizable fields that are used to expand and define ksonnet
```

Comment:
I'm not sure `expand` would be the right terminology here. Would prefer to leave it at `define....`